app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,18 @@
 <resources>
     <string name="app_name">Weather</string>
+    <string name="title_ok">OK</string>
+    <string name="text_o">o</string>
+    <string name="title_permission_denied">Permission denied</string>
+    <string name="title_permission_rationale">Location permission is needed for core functionality</string>
+    <string name="title_permission_denied_explanation">Permission was denied, but is needed for core
+        functionality.</string>
+    <string name="title_today">TODAY</string>
+    <string name="title_senvenday">7 DAY</string>
+    <string name="title_no_internet">No connection internet!</string>
+    <string name="title_location_unavailable">Location unavailable!</string>
+    <string name="title_visibality">Visibality</string>
+    <string name="title_humidity">Humidity</string>
+    <string name="title_windspeed">Windspeed</string>
+    <string name="title_pressure">Pressure</string>
+    <string name="title_current_detail">Current Detail</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_current_detail` appears to be unused
 Unused resources make applications larger and slow down builds.